src/Formatting.java

```diff
@@ -16,7 +16,7 @@ public static void format() {
             FileWriter formattedWrite = new FileWriter(formated);
             while (sc.hasNext()) {
                 //split userdata into 11 elements
-                String topSpace = String.format("%83s","").replace(' ','=');
+                String topSpace = String.format("%96s","").replace(' ','=');
```

Comment:
??? what does this do? "%96s"?

Reply:
creates an empty string that is 96 whitespaces long and then replaces it with "=" to make it look nice and ticket like.